src/meta/src/hummock/manager/mod.rs

```diff
@@ -887,7 +886,7 @@ impl HummockManager {
                     .sum::<usize>(),
                 start_time.elapsed()
             );
-        } else if is_trivial_move && can_trivial_move {
```

Comment:
Is it because all trivial move tasks are `TaskType::Dynamic` that we can remove this check?

Reply:
IMO, we don't actually need to disable trivial-move, regardless of the type of task?